src/labels/label_point.js

```diff
@@ -50,8 +51,8 @@ export default class LabelPoint extends Label {
     }
 
     updateBBoxes () {
-        let width = (this.size[0] + this.layout.buffer[0] * 2) * this.layout.units_per_pixel * Label.epsilon;
-        let height = (this.size[1] + this.layout.buffer[1] * 2) * this.layout.units_per_pixel * Label.epsilon;
```

Comment:
I think this epsilon was originally added to prevent objects that were "perfectly" aligned (e.g. a `10px` sprite with a label positioned exactly `10px` below) from colliding one another. It may not be needed anymore (since then there have been explicit sprite/text label linking code added, and collision code changes), but we will need to look closely.